pkgs/tools/audio/beets/common.nix

```diff
@@ -154,10 +144,11 @@ python3Packages.buildPythonApplication {
   nativeCheckInputs =
     with python3Packages;
     [
-      pytestCheckHook
-      pytest-cov
       mock
+      pytest-cov
+      pytestCheckHook
```

Comment:
I don't mind too much changes of order in inputs, but I'd like these to be minimized if they were random, or if they are justified then be put in a separate commit.

Reply:
solely to alpha sort them, I can break out into a separate commit or just undo if you don't want them.